src/Domain/DomainCreators/Interval.cpp

```diff
@@ -22,14 +22,14 @@ Interval::Interval(
     const typename IsPeriodicIn::type is_periodic_in_x,
     const typename InitialRefinement::type initial_refinement_level_x,
     const typename InitialGridPoints::type initial_number_of_grid_points_in_x,
-    const OptionContext& /*context*/)
```

Comment:
While you're here, might as well make these non-const and move (remember clang-tidy)